deploy/helm-operator-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+---
+  apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: fluxhelmresources.integrations.flux
+  spec:
+    group: integrations.flux
+    names:
+      kind: FluxHelmResource
```

Comment:
Everything is a resource, so that doesn't need to be in the name. "Flux" is in the API group, but commands and code don't always (or even) refer to that, so probably worth keeping to avoid clashes. 
`Chart` is more specific than `Helm`, unless you reckon any later code dealing with packages will use this definition too. Lastly, probably worth having `Release` in the name, since that what the resources will represent. So, either `FluxChartRelease` or `FluxHelmRelease`, I reckon.

Reply:
I think FluxHelmRelease is a good name

Reply:
done